components/BiolucidaViewer/BiolucidaViewer.vue

```diff
@@ -43,6 +47,24 @@ export default {
         }
       }
     }
+  },
+  watch: {
+    queryView() {
+      this.$refs.biolucida.contentWindow.postMessage('getImgPos')
+    }
+  },
+  mounted() {
+    window.addEventListener('message', this.receiveMessage)
```

Comment:
I'm guessing Biolucida's image viewer only communicates through window message events?

Reply:
This is for the postMessage messaging between the portal and Biolucida.